src/auro-pane.js

```diff
@@ -3,29 +3,57 @@
 
 // ---------------------------------------------------------------------
 
-import { LitElement, html, css } from "lit-element";
+import { LitElement, html, css } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
 
 // Import touch detection lib
-import "focus-visible/dist/focus-visible.min.js";
-import styleCss from "./style-css.js";
+import 'focus-visible/dist/focus-visible.min.js';
+import styleCss from './style-css.js';
 
 // See https://git.io/JJ6SJ for "How to document your components using JSDoc"
 /**
  * auro-pane provides users a way to ...
  *
- * @attr {String} cssClass - Applies designated CSS class to DOM element.
+ * @attr {String} date - Sets date for display. Format should be yyyy-mm-dd.
+ * @attr {Boolean} disabled - Disables the pane and overrides price to be --.
+ * @attr {Boolean} selected - Sets pane state to selected.
  */
 
-// build the component class
 class AuroPane extends LitElement {
-  // constructor() {
-  //   super();
-  // }
+  constructor() {
+    super();
+
+    this.weekdays = [
+      'Sun',
+      'Mon',
+      'Tue',
+      'Wed',
+      'Thu',
+      'Fri',
+      'Sat'
+    ];
+
+    this.months = [
+      'Jan',
+      'Feb',
+      'Mar',
+      'Apr',
+      'May',
+      'Jun',
+      'Jul',
+      'Aug',
+      'Sep',
+      'Oct',
+      'Nov',
+      'Dec'
+    ];
+  }
 
-  // function to define props used within the scope of thie component
   static get properties() {
     return {
-      cssClass:   { type: String }
+      date: { type: String },
```

Comment:
For a more generic pane, would it make sense to change this to a "Primary" string and also have a secondary prop (for my fare display)?

Reply:
We would need three props then for each of the text displays ("Wed", "Sep 10", "$233"). I don't think we'd be able to do the date parsing then either.

Do we think this is going to be used in more generic situations? I had assumed this would only be used to display shoulder dates with or without price.

Reply:
So, what I am seeing here is that there is an opportunity for fixed or anonymous content. See comments below ...

Reply:
I don't think we have a current use case for more generic situations, but agreed on the primary/secondary (date/price).

Reply:
@gusnaughton to clarify, do you want me to replace "date" and "price" with "primary", "secondary", and "tertiary" strings and remove the date parsing? Or leave it as is?

I'm in favor of keeping date and price props to make the intent clearer and standardize the date formatting cross-experience until there's a need for a more generic use. The CSS would also have to be further optimized if we're expecting any string to display instead of assuming a 3-character weekday + 6-character date. I'm willing to change it if you think it's important, though.

Reply:
I agree with what you're saying. I like the idea of a price prop, we just need to make sure that it can wrap to the next line if it's a miles+money deal (example: 20k + $5.60). A slot might make sense for this, but open to any implementation.